package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "chai": "^4.1.0",
```

Comment:
Yeah I don't think you need either of these. Please just use jest.fn for mocking